src/families/ethereum/bridge/mock.js

```diff
@@ -25,15 +26,15 @@ const receive = makeAccountBridgeReceive();
 const defaultGetFees = (a, t: *) =>
   (t.gasPrice || BigNumber(0)).times(getGasLimit(t));
 
-const createTransaction = (account): Transaction => ({
+const createTransaction = (): Transaction => ({
   family: "ethereum",
   mode: "send",
   amount: BigNumber(0),
   recipient: "",
   gasPrice: BigNumber(10000000000),
   userGasLimit: BigNumber(21000),
   estimatedGasLimit: null,
-  feeCustomUnit: account.currency.units[1],
+  feeCustomUnit: getCryptoCurrencyById("ethereum").units[1],
```

Comment:
this shouldn't be needed. we can't mock ethereum classic tx because of this change so not fan of it.